edge-modules/api-proxy-module/rust-sdk/hyper-uds/src/lib.rs

```diff
@@ -184,22 +184,23 @@ impl hyper::client::connect::Connection for UnixStream {
 }
 
 impl tokio::io::AsyncRead for UnixStream {
-	fn poll_read(self: std::pin::Pin<&mut Self>, cx: &mut std::task::Context<'_>, buf: &mut [u8]) -> std::task::Poll<std::io::Result<usize>> {
+	fn poll_read(self: std::pin::Pin<&mut Self>, cx: &mut std::task::Context<'_>, buf: &mut tokio::io::ReadBuf<'_>) -> std::task::Poll<std::io::Result<()>> {
```

Comment:
I'm confused.

1. This crate isn't needed. It only existed to [provide Windows support for UDS.](https://github.com/arsing/hyper-uds/blame/master/README.md#L3)

1. This copy of the crate seems to have taken the inefficient Windows impl that was originally in [windows.rs](https://github.com/arsing/hyper-uds/blob/master/src/windows.rs), moved it into `lib.rs`, and is using it even for Linux. Why?!

Reply:
we can ask author of this module... @huguesBouvier do you remember why you did this change?

Reply:
I removed `hyper-uds` altogether in favor of `hyperlocal`